pinning.go

```diff
@@ -152,52 +170,65 @@ func (cm *ContentManager) refreshPinQueue() error {
 	// anyways
 	makeDeal := true
 	for _, c := range toPin {
+		var origins []*peer.AddrInfo
+		// when refreshing pinning queue, use content origins if available
+		if c.Origins != "" {
+			_ = json.Unmarshal([]byte(c.Origins), &origins)
```

Comment:
Handle here
```
if err := json.Unmarshal([]byte(c.Origins), &origins)); err != nil {
        log.Warnf("Content origins is invalid: %s", err)
}
```

Reply:
No need to handle or log the error - it is not actionable, using the origin here is a nice to have. I have added a comment to state why we don't need to